rules/BUILD

```diff
@@ -2,4 +2,11 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
+load("//rules:opentitan.bzl", "OPENTITAN_PLATFORM")
+
 package(default_visibility = ["//visibility:public"])
+
+config_setting(
+    name = "opentitan_platform",
+    values = {"platforms": OPENTITAN_PLATFORM},
+)
```

Comment:
Is there any reason this cannot live in `rules/opentitan.bzl`? Seems more appropriate to be defined there?

Reply:
It can't. You can't define targets in .bzl files.